.github/workflows/gpu_signed_integer_classic_tests.yml

```diff
@@ -44,6 +44,9 @@ jobs:
               - tfhe/build.rs
               - backends/tfhe-cuda-backend/**
               - tfhe/src/core_crypto/gpu/**
+              - tfhe/src/integer/server_key/radix_parallel/tests_unsigned/**
```

Comment:
Why unsigned tests in this case ?

Reply:
No we have to be sure the GPU CI gets triggered in case any test changes on the CPU side, since the GPU tests actually use functions from those files. The functions are in the unsigned & signed tests folders, and in the tests_cases_unsigned.rs file.

Reply:
Thanks for the explanation.